build/lib/gradio/inputs.py

```diff
@@ -417,6 +449,15 @@ def get_interpretation_scores(self, x, neighbors, scores):
             final_scores.append(score_set)
         return final_scores
 
+    def embed(self, x):
+        if self.type == "value":
+            return [choice in x for choice in self.choices]
```

Comment:
shouldn't this be [float(choice in x) for choice in self.choices]

Reply:
Not strictly necessary, since python does the implicit casting, but I'll make it explicit